src/main/kotlin/blackjack/domain/Rule.kt

```diff
@@ -4,17 +4,22 @@ import blackjack.domain.model.Dealer
 import blackjack.domain.model.Player
 
 object Rule {
+    const val DEALER_MINIMUM_SCORE = 17
+    const val BLACK_JACK = 21
+
     fun decisionWinner(dealer: Dealer, player: Player): Player? {
         val dealerSum = dealer.cards.sum
         val playerSum = player.cards.sum
 
-        if (isOverBlackJack(playerSum)) return dealerWin(dealer, player)
-
         if (isOverBlackJack(dealerSum)) return playerWin(dealer, player)
 
-        return if (dealerSum > playerSum) dealerWin(dealer, player)
-        else if (dealerSum < playerSum) playerWin(dealer, player)
-        else null
+        if (isOverBlackJack(playerSum)) return dealerWin(dealer, player)
+
+        return when {
+            dealerSum > playerSum -> dealerWin(dealer, player)
+            dealerSum < playerSum -> playerWin(dealer, player)
+            else -> null
+        }
```

Comment:
```suggestion
        if (dealerSum > playerSum) {
            dealerWin(dealer, player)
        } else {
            dealerSum < playerSum -> playerWin(dealer, player)
        }
```
블랙잭에서는 딜러가 이기거나 혹은 플레이어가 이기거나 두 가지 경우밖에 없습니다.
그렇다면 if-else문을 활용해보면 어떨까요?
이렇게 되면 리턴타입을 non-null로 변경할 수 있을 것 같아요!

추가로 코틀린에서는 조건이 두 가지인 경우에는 if-else문을, 3가지 이상인 경우에는 when을 사용하는 것을 권장합니다.
ref. https://kotlinlang.org/docs/coding-conventions.html#if-versus-when

Reply:
딜러와 플레이어가 동점인 경우에 어떻게 처리하는지 명세에 나와있지 않아서 규칙을 찾아보았는데, 딜러와 플레이어의 카드의 합이 같은 경우에는 승부가 나지 않는다고 해서 `Rule.decisionWinner` 에서 무승부가 난 경우에는 `null`을 리턴하도록 구현했습니다. 
이 부분은 제가 잘못 구현한건가요? 
규칙에 대해 생략된 부분이 많아서 헷갈리네요😅

Reply:
앗 제가 무승부인 경우를 놓쳤네요. 😭 
혼란을 드려 죄송합니다. 😭 
이 부분은 무시해주세요. 🙏